umh.docs.umh.app/content/en/docs/production-guide/installation/flatcar-installation-bare-metal.md

```diff
@@ -13,106 +13,162 @@ aliases:
 
-## Install Flatcar Linux on the edge device
 
-1. Connect the USB stick to the edge device and boot it. Each device has a
-   different way of booting from USB, so you need to consult the documentation
-   of your device.
+
+<!-- steps -->
+## Installation Steps
+
+1. If necessary, boot the downloaded ISO image. For bare metal, connect the USB stick to the device and boot it. Each device has a different way of booting from USB, so you need to consult your device's documentation.  
 2. Accept the License.
+![Untitled](/images/production-guide/flatcar-installation/flatcar1.png)
+
 3. Select the correct network settings. If you are unsure, select DHCP, but
    keep in mind that a static IP address is strongly recommended.
+   ![Untitled](/images/production-guide/flatcar-installation/flatcar2.png)
+   ![Untitled](/images/production-guide/flatcar-installation/flatcar3.png)
+   ![Untitled](/images/production-guide/flatcar-installation/flatcar4.png)
+
 4. Select the correct drive to install Flatcar Linux on. If you are unsure, check
    the [troubleshooting section](#drive).
-5. Check that the installation settings are correct and press `Confirm` to start
+  ![Untitled](/images/production-guide/flatcar-installation/flatcar5.png)
+
+5. Check that the installation settings are correct, and press **Confirm** to start
    the installation.
+  ![Untitled](/images/production-guide/flatcar-installation/flatcar6.png)
+  
+
+6. Now, the installation will start. 
+  ![Untitled](/images/production-guide/flatcar-installation/flatcar7.png)
 
-Now the installation will start. You should see a green command line soon after,
-that says `core@flatcar-0-install ~$~`. Now remove the USB stick from the
-device. At this point the system is still installing. After a few minutes,
-depending on the speed of your network, the installation will finish and the
-system will reboot. Now you should see a grey login prompt that says
+7. Shortly after, you should be able to see a green command line `core@flatcar-0-install ~$~`. Then, remove the USB stick from the
+device. At this point, the system is still processing the installation. After a few minutes,
+depending on the speed of your network, the installation will finish, and the
+system will reboot.
+  ![Untitled](/images/production-guide/flatcar-installation/flatcar8.png)
+
+8. Now you should see a grey login prompt that says
 `flatcar-1-umh login:`, as well as the IP address of the device.
+  ![Untitled](/images/production-guide/flatcar-installation/flatcar9.png)
+
 
 {{% notice note %}}
 Please note that the installation may take some time. This largely depends on the available resources
 including network speed and system performance.
 {{% /notice %}}
 
-## Connect to the edge device
+### After iPXE (**Proxmox**)
+1. After the first start of the VM, remove the auto boot option in Proxmox. Go to **Hardware**, double click on **CD/DVD DRIVE** and select **Do not use any media**.
```

Comment:
i don't think is is needed, I never needed to do this.

maybe we can generalize this also? we could say "a lot of system will by default boot from hard disk However, sometimes they are configured to boot always from DVD/ USB-stick, so if you keep seeing the iPXE screen (even after successful installation)